Earthfile

```diff
@@ -1,72 +0,0 @@
-VERSION 0.6
-FROM mcr.microsoft.com/playwright:v1.34.0-jammy
-
-WORKDIR /usr/build
-
-build:
-  COPY . .
-  RUN yarn --immutable && yarn run build
-
-typecheck:
-  FROM +build
-  RUN yarn run typecheck
-
-lint:
-  FROM +build
-  RUN yarn run lint
-
-test:
-  FROM +build
-  RUN yarn run test
-
-setup:
-  FROM +build
-  RUN apt update && apt install xz-utils
-  # it's added here to deps to avoid affecting Turborepo's dependency graph and related caching
-  RUN yarn add --dev "replayio@workspace:^" && npx --no replayio install
-  # download binary openssl packages from Impish builds
-  RUN wget http://security.ubuntu.com/ubuntu/pool/main/o/openssl/openssl_1.1.1f-1ubuntu2.22_amd64.deb
-  RUN wget http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl-dev_1.1.1f-1ubuntu2.22_amd64.deb
-  RUN wget http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.22_amd64.deb
-  # install downloaded binary packages
-  RUN dpkg -i libssl1.1_1.1.1f-1ubuntu2.22_amd64.deb
-  RUN dpkg -i libssl-dev_1.1.1f-1ubuntu2.22_amd64.deb
-  RUN dpkg -i openssl_1.1.1f-1ubuntu2.22_amd64.deb
-
-flake:
```

Comment:
I have not reimplemented this part here. It's somewhat complicated and we don't rely on this at the moment anyway